x/staking/types/hooks_template.go

```diff
@@ -0,0 +1,32 @@
+package types
```

Comment:
can this file be removed?

Reply:
@jtremback?

Reply:
It contains a ` StakingHooksTemplate` struct that is used on the provider to register multiple staking hooks.

https://github.com/cosmos/interchain-security/blob/ad4231d80669705edef0a62524e6b5c8bd50d4f1/x/ccv/provider/keeper/keeper.go#L512-L515